cmd/snap/cmd_snap_op.go

```diff
@@ -201,8 +209,33 @@ func (x *cmdRefresh) Execute([]string) error {
 	return listSnaps([]string{name})
 }
 
+type cmdTry struct {
+	DevMode    bool `long:"devmode" description:"Install in development mode and disable confinement"`
```

Comment:
@jdstrand Thanks! I added a bunch of integration tests to ensure that the confinement works. The try-dir is bind mounted so from the confinement perspective it should be exactly the same as a squashfs mount. However, thanks for raising it, I was thinking about it and wrote a bunch of tests to ensure this is really the case (and that we don't regress here :)